src/routing/Router.tsx

```diff
@@ -19,15 +21,19 @@ import { RouterContext, RouterContextProvider } from './RouterContext';
  */
 export default function FormRouter(props: RouterProps): JSX.Element {
   const initialValues = props.formData;
+  const [route, updateRoute] = useState('/');
 
   return (
     <div className="row">
       <div className="usa-width-two-thirds medium-8 columns">
-        <RouterContextProvider routes={props.children}>
-          <BrowserRouter basename={props.basename}>
-            {props?.title && (
-              <FormTitle title={props.title} subTitle={props?.subtitle} />
-            )}
+        <BrowserRouter basename={props.basename}>
```

Comment:
*Not a code change* but was there a reason to move the context under the router?

Reply:
no, it was just a stylistic choice. It doesn't really change anything either way but might need to be reordered if we go back and use history to listen for router changes.